packages/shared-components/src/components/onto-toastr/onto-toastr.scss

```diff
@@ -0,0 +1,60 @@
+.onto-toastr-container {
+  position: fixed;
+  z-index: 9999999;
+  pointer-events: auto;
+  flex-direction: column;
+
+  &.bottom-right {
+    bottom: 12px;
+    right: 12px;
+  }
+
+  &.bottom-left {
+    bottom: 12px;
+    left: 12px;
+  }
+
+  & .onto-toast {
```

Comment:
Also consider that if you have a longer unbreakable string, the toastr won't render it properly
![image](https://github.com/user-attachments/assets/de135357-7829-4724-9577-4812d8aafa0f)